airbyte-integrations/bases/debezium-v1-9-2/src/main/java/io/airbyte/integrations/debezium/internals/PostgresConverter.java

```diff
@@ -39,9 +43,50 @@ public void converterFor(final RelationalColumn field, final ConverterRegistrati
       registerText(field, registration);
     } else if (Arrays.stream(MONEY_ITEM_TYPE).anyMatch(s -> s.equalsIgnoreCase(field.typeName()))) {
       registerMoney(field, registration);
+    } else if (BYTEA_TYPE.equalsIgnoreCase(field.typeName())) {
+      registerBytea(field, registration);
+    } else if (Arrays.stream(NUMERIC_TYPES).anyMatch(s -> s.equalsIgnoreCase(field.typeName()))) {
+      registerNumber(field, registration);
     }
   }
 
+  private void registerNumber(final RelationalColumn field, final ConverterRegistration<SchemaBuilder> registration) {
+    registration.register(SchemaBuilder.string().optional(), x -> {
+      if (x == null) {
+        return DebeziumConverterUtils.convertDefaultValue(field);
+      }
+      // Bad solution
+      // We applied a solution like this for several reasons:
+      // 1. Regarding #13608, CDC and nor-CDC data output format should be the same.
+      // 2. In the non-CDC mode 'decimal' and 'numeric' values are put to JSON node as BigDecimal value.
+      // According to Jackson Object mapper configuration, all trailing zeros are omitted and
+      // numbers with decimal places are deserialized with exponent. (e.g. 1234567890.1234567 would
+      // be deserialized as 1.2345678901234567E9).
+      // 3. In the CDC mode 'decimal' and 'numeric' values are deserialized as a regular number (e.g.
+      // 1234567890.1234567 would be deserialized as 1234567890.1234567). Numbers without
+      // decimal places (e.g 1, 24, 354) are represented with trailing zero (e.g 1.0, 24.0, 354.0).
+      // One of solution to align deserialization for these 2 modes is setting
+      // DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS as true for ObjectMapper. But this breaks
+      // deserialization for other data-types.
+      // A worked solution was to keep deserialization for non-CDC mode as it is and change it for CDC
+      // one.
+      // The code below strips trailing zeros for integer numbers and represents number with exponent
+      // if this number has decimals point.
+      final double doubleValue = Double.parseDouble(x.toString());
+      var valueWithTruncatedZero = BigDecimal.valueOf(doubleValue).stripTrailingZeros().toString();
+      return valueWithTruncatedZero.contains(".") ? String.valueOf(doubleValue) : valueWithTruncatedZero;
+    });
+  }
+
+  private void registerBytea(final RelationalColumn field, final ConverterRegistration<SchemaBuilder> registration) {
```

Comment:
There is a property [binary.handling.mode](https://debezium.io/documentation/reference/stable/connectors/postgresql.html#postgresql-property-binary-handling-mode) that we can use 

```
Specifies how binary (bytea) columns should be represented in change events:

bytes represents binary data as byte array.

base64 represents binary data as base64-encoded strings.

hex represents binary data as hex-encoded (base16) strings.
```